src/Schema.js

```diff
@@ -802,17 +804,23 @@ const makeDescriptor = function (prop, schema, opts) {
     // TODO: Make it so tracking can be turned on for all properties instead of
     // only per-property
     if (track && !_get(creatingPath)) {
+      /* previous is versioned on database commit
+       * props are versioned on set()
+       * TODO? Option to disable props versioning (commit only versioning)
```

Comment:
No. The change history, played backwards on the records, should produce the value contained in `previous`. Once a record is saved or committed, previous and change history reset, as they're meant to represent in-memory (un-persisted changes). If the user wants to keep a history of changes that happened _before_ a save/commit, then they'll need to save them somewhere else (which someone might do if they're implementing some kind of "undo" functionality in a UI, that reverses changes that have already been persisted to the database).